pkgs/development/libraries/discord-gamesdk/default.nix

```diff
@@ -14,6 +15,10 @@ stdenv.mkDerivation rec {
     stripRoot = false;
   };
 
+  outputs = [ "out" "dev" ];
+
+  buildInputs = [ gcc.cc.libgcc ];
```

Comment:
```suggestion
  buildInputs = [ (stdenv.cc.cc.libgcc or null) ];
```
`buildInputs` should be below `nativeBuildInputs`

Reply:
Done, except wdym "remember to remove the input"? Which input, where?

Reply:
Done.